show/main.py

```diff
@@ -1,20 +1,20 @@
 #! /usr/bin/python -u
 
 import json
-import netaddr
+#import netaddr
 import os
 import subprocess
 import sys
 
 import click
 from natsort import natsorted
 import netifaces
-from pkg_resources import parse_version
+#from pkg_resources import parse_version
 
 import feature
 import interfaces
 import kube
-import mlnx
+#import mlnx
```

Comment:
I believe commenting this out completely will break Mellanox platforms.

Reply:
this is included after checking for mellanox platform. see below (line 1008)